docs/lib/content/configuring-npm/package-json.md

```diff
@@ -285,6 +285,7 @@ Certain files are always included, regardless of settings:
 * `README`
 * `LICENSE` / `LICENCE`
 * The file in the "main" field
+* The file in the "bin" field
```

Comment:
```suggestion
* The file(s) in the "bin" field
```
there can be multiple if it’s an object.

Reply:
that's right!